src/FSharpLint.Core/Rules/Conventions/SuggestUseAutoProperty.fs

```diff
@@ -0,0 +1,99 @@
+﻿module FSharpLint.Rules.SuggestUseAutoProperty
+
+open System
+open FSharpLint.Framework
+open FSharpLint.Framework.Suggestion
+open FSharp.Compiler.Syntax
+open FSharpLint.Framework.Ast
+open FSharpLint.Framework.Rules
+
+let rec private isImmutableValueExpression (args: AstNodeRuleParams) (expression: SynExpr) =
+    match expression with
+    | SynExpr.Const (constant, range) -> true
```

Comment:
@webwarrior-ws please add _ prefix to unused vars

Reply:
done